ompi/mca/osc/base/osc_base_init.c

```diff
@@ -74,6 +74,109 @@ ompi_osc_base_select(ompi_win_t *win,
     opal_output_verbose( 10, ompi_osc_base_framework.framework_output,
                          "select: component %s selected",
                          best_component->osc_version.mca_component_name );
-
+    
     return best_component->osc_select(win, base, size, disp_unit, comm, info, flavor, model);
+    
+}
+
+int
+ompi_osc_base_iselect(ompi_win_t *win,
```

Comment:
I'm not sure this is needed - the "select" is choosing which module to use for supporting the RMA functionality. It is already a local (nonblocking) action, I think. So I'm pretty sure we don't have to split it into a begin piece and a complete piece.